jinseok/service/likeService.js

```diff
@@ -0,0 +1,24 @@
+const likeDao = require('../model/likeDao');
+
+const createDeleteLike = async (userId, postId) => {
+  let createOrDelete;
+  const like = await likeDao.getLike(userId, postId);
+  if (!(JSON.stringify(like) === '[]')) {
```

Comment:
```suggestion
  const isLiked = await likeDao.getLike(userId, postId);
  if (!isLiked) {
```
위의 코멘트를 반영하면 다음과 같이 부정연산자로 분기처리를 할 수 있습니다.